src/inputs/durationPicker/__test__/durationPicker.test.js

```diff
@@ -0,0 +1,622 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
+ * npx jest ./src/inputs/durationPicker/__test__/durationPicker.test.js
+ */
+import { mount, shallow } from 'enzyme';
+import moment from 'moment-timezone';
+import React from 'react';
+import { BEM_BLOCK_NAME } from '../constants';
+import DurationPicker from '../durationPicker';
+import StyledDurationPicker from '../styledDurationPicker';
+import MockedTheme from '../../../testUtils/mockedTheme';
+
+describe('<DurationPicker />', () => {
+    it('Can render without problems (all default props)', () => {
+        const wrapper = mount(
+            <MockedTheme>
```

Comment:
Let's merge my #192 PR and then update mount methods in this file.

Reply:
Yep, will do

Reply:
Fixed in 629045a.